### PR TITLE
#132 Remove invoice id from line mutations

### LINKED ISSUE #132: Remove invoice id from line mutations
Remove invoice id from delete and update line mutations and reflect this change in frontend

### PR DESCRIPTION
**Pull request by roxy-dao**
Closes #132

- Deleted invoice id from update and delete invoice line mutations.

### CODE REVIEW

File: server/service/src/invoice_line/outbound_shipment_service_line/update/mod.rs [hunk 1]
```diff
@@ -133,45 +127,42 @@ mod test {
...
                 inline_init(|r: &mut UpdateOutboundShipmentServiceLine| {
-                    r.invoice_id = mock_draft_inbound_shipment_with_service_lines().id;
                     r.id = mock_draft_inbound_service_line().id;
                 }),
             ),
             Err(ServiceError::NotAnOutboundShipment)
         );
 
         // NotThisInvoiceLine
-        assert_eq!(
```
Comment (andreievg) [inline server/service/src/invoice_line/outbound_shipment_service_line/update/mod.rs:162]: happy for you to remove these tests

**Review by andreievg — COMMENTED** [review]
I haven't looked in depth but i see update in server for: update/delete outbound shipment, update/delete for inbound shipment, and can only see deletes touched in client

**Comment by roxy-dao** [comment]
> I haven't looked in depth but i see update in server for: update/delete outbound shipment, update/delete for inbound shipment, and can only see deletes touched in client

It's in the api.ts files 🤔

File: server/service/src/invoice_line/inbound_shipment_service_line/delete/validate.rs [hunk 2]
```diff
@@ -19,9 +17,9 @@ pub fn validate(
     connection: &StorageConnection,
 ) -> Result<InvoiceLineRow, DeleteInboundShipmentServiceLineError> {
     let line = check_line_exists(&input.id, connection)?;
-    let invoice = check_invoice_exists(&input.invoice_id, connection)?;
+    let invoice = check_invoice_exists(&line.invoice_id, connection)?;
 
-    check_line_belongs_to_invoice(&line, &invoice)?;
+    // check_line_belongs_to_invoice(&line, &invoice)?;
```
Comment (clemens-msupply) [inline server/service/src/invoice_line/inbound_shipment_service_line/delete/validate.rs:22]: Remove...

I might miss something but has check_line_belongs_to_invoice been removed, is it still needed?

**Review by clemens-msupply — APPROVED** [review]
Looks good to me thank you

**Review by andreievg — APPROVED** [review]

**Pull request merged**